test/starknet/shared/types.ts

```diff
@@ -1,5 +1,9 @@
 import { assert, hexToBytes, bytesToHex } from './helpers';
 
+export const AGAINST = BigInt(1);
```

Comment:
If we can't use 0 for "Against", it's better to have the choice in the order it will be render on UI like this:
1: For
2: Against
3: Abstain

Reply:
Ok! Was following the order of `choice.cairo`. I've updated everything at the same time in https://github.com/snapshot-labs/sx-core/pull/47/commits/355c2f92425bdcdc43a4d6d6b76b91e8b485e373 :)